### PR TITLE
Finished Activity 5

### PR DESCRIPTION
**Pull request by iefomit**
## Overview

This PR will display the PID Controller in Shuffleboard, and run SubsystemPIDCommand on Buttons START and BACK.

## Activity Finished

Finished Activity 5

## Tests Ran

Tested & Tuned PID Controller, Tested PID Command on Button Presses.

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/DriveSub.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+                    0); // disable gravity
+        }
+
+        // initialize mechanism2d and ligament
+        mech2d = new Mechanism2d(100, 100);
+        wheelLigament = new MechanismLigament2d("Wheel", 30, 0);
+
+        // intialize pid
+        pidController = new PIDController(0.03, 0, 0.11);
+        pidController.setTolerance(0.05);
```
Comment (Kyle-Eldridge) [inline src/main/java/frc/robot/subsystems/DriveSub.java:46]: These values get the subsystem to a position accurately but not quickly. Try tuning more to get to the setpoint both quickly and accurately.

**Review by Kyle-Eldridge — CHANGES_REQUESTED** [review]
The code looks good, but the PID could be tuned more to go faster. Also, since this is the last activity we're requiring code for, we might re-review all of your code soon.

File: src/main/java/frc/robot/subsystems/DriveSub.java (hunk 1 above)
Comment (iefomit) [reply src/main/java/frc/robot/subsystems/DriveSub.java:46]: Just tuned it. Might be too fast, though seems fairly accurate.

**Review by Kyle-Eldridge — APPROVED** [review]

**Comment by Kyle-Eldridge** [comment]
You have finished all of the required code for software training. If you want to practice using feedforward after we present it tomorrow, you can optionally do whichever parts of Activity 6 you want to do. Alternatively, you could read and try to understand our 2024 code on the FRC-2024 repository. If you do this, you should start with the subsystems (except drivetrain) and commands. The drivetrain and vision sections are probably the most complicated, so you can choose whether or not you skip them.

**Pull request closed without merging**